src/web/mjs/connectors/SodaScan.mjs

```diff
@@ -0,0 +1,12 @@
+import WordPressMadara from './templates/WordPressMadara.mjs';
+
+export default class SodaScan extends WordPressMadara {
+
+    constructor() {
+        super();
+        super.id = 'sodascan';
+        super.label = 'Soda Scan';
+        this.tags = [ 'manga', 'webtoon', 'portuguese', 'scanlation' ];
+        this.url = 'https://sodascan.xyz/';
```

Comment:
May remove trailing slash `/`, why this don't cause an error it would be good practice to be consistent with the existing code that expects no trailing slash on `this.url`